queries/models/projeto_subsidio_sppo/CHANGELOG.md

```
@@ -1,5 +1,12 @@
 # Changelog - projeto_subsidio_sppo
 
+## [9.3.5] - 2025-09-30
+
+### Alterado
+
+- Ajusta o modelo efêmero `viagem_planejada_v1` removendo o trecho `incremental`.(https://github.com/prefeitura-rio/pipelines_rj_smtr/pull/907)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)